code/src/components/HandleTODO.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable max-len */
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import tickets from 'reducers/tickets';
+import { HandleTODOButton } from './buttons';
+
+// we pass down item since the dispatches regard updates to single items, and eith "item" we add the information about which one it is
```

Comment:
Really nice that the user can take actions on individual tasks. Def   stretch goals.